contracts/middle-layer/resource-mirror/ObjectHub.sol

```diff
@@ -12,13 +12,17 @@ import "../../interface/IObjectHub.sol";
 contract ObjectHub is ObjectStorage, GnfdAccessControl, CmnHub, IObjectHub {
     using DoubleEndedQueueUpgradeable for DoubleEndedQueueUpgradeable.Bytes32Deque;
 
+    /*----------------- initializer -----------------*/
     function initialize(address _ERC721_token, address _additional) public initializer {
-        ERC721Token = _ERC721_token;
-        additional = _additional;
+        __cmn_hub_init_unchained(_ERC721_token, _additional);
 
         channelId = OBJECT_CHANNEL_ID;
     }
 
+    function initializeV2() public reinitializer(2) {
```

Comment:
The deployer should call `initializeV2`, dont they?

Reply:
the `Deployer` contract will call initializeV2 if it is deployed in the first time.